<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BmMonitorMainWindow</class>
 <widget class="QMainWindow" name="BmMonitorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Brandmeister Monitor by UT8UU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54); color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Talk Groups</string>
     </attribute>
     <widget class="QCheckBox" name="cbTalkGroups">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>190</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use groups</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbDisplayAllGroups">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>190</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Display all groups</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListView" name="lbGroups">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>250</width>
        <height>410</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Callsigns</string>
     </attribute>
     <widget class="QCheckBox" name="cbCallsigns">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use callsigns</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tbCallsigns">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>250</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabRecords">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAllRecords">
     <attribute name="title">
      <string>All records</string>
     </attribute>
     <widget class="QListView" name="monitoringList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>470</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFilteredRecords">
     <attribute name="title">
      <string>Filtered records</string>
     </attribute>
     <widget class="QComboBox" name="cbTimestampFilter">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>10</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Last hour</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Last 6 hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Last 12 hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Last 24 hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Last week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>20</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Filter:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="tbFilterCallsign">
      <property name="geometry">
       <rect>
        <x>371</x>
        <y>10</y>
        <width>190</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QListView" name="monitoringFilteredList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
